stack.yaml

```diff
@@ -1,5 +1,6 @@
 flags: {}
 nix:
+  enable: true
```

Comment:
oops I did it again; good thing I went PR way for this reason this time, will fix